toucan_connectors/mysql/mysql_connector.py

```diff
@@ -82,7 +82,7 @@ def get_form(cls, connector: 'MySQLConnector', current_config: dict[str, Any]) -
     # Columns from the columns table
     'c.column_name, c.data_type FROM information_schema.tables t INNER JOIN information_schema.columns c '
     # Inner join on table name
-    'ON t.table_name = c.table_name '
+    'ON t.table_name = c.table_name AND t.table_schema = c.table_schema '
```

Comment:
Maybe a unit test for this change could be great ?
what do you think ?

Reply:
c'est vrai, mais pour ça il faudrait setup deux schemas dans le mysql, et j'ai la flemme :upside_down_face: 
Sachant que ça a été testé en prod et que c'est la même logique que pour les autres dbms, cf. https://github.com/ToucanToco/toucan-connectors/pull/574

Reply:
all good si le test en prod est positif, c'est pourquoi j'ai approve 👌🏽